src/include/optimizer/operators.h

```diff
@@ -179,10 +179,10 @@ class LogicalDelete : public OperatorNode<LogicalDelete> {
 class LogicalUpdate : public OperatorNode<LogicalUpdate> {
  public:
   static Operator make(storage::DataTable *target_table,
-                       std::vector<peloton::parser::UpdateClause*> updates);
+                       std::vector<std::unique_ptr<peloton::parser::UpdateClause>>* updates);
```

Comment:
You changed this from a "vector of raw pointers" to a "pointer to a vector of unique pointers". Why? Can we just pass the vector by reference?